std/net/curl.d

```diff
@@ -2330,9 +2330,9 @@ private bool decodeLineInto(Terminator, Char = char)(ref const(ubyte)[] basesrc,
   * http.perform();
   *
   * // Track progress
+  * auto http = HTTP();
   * http.method = HTTP.Method.get;
-  * http.url = "http://upload.wikimedia.org/wikipedia/commons/"
-  *            "5/53/Wikipedia-logo-en-big.png";
+  * http.url = "http://upload.wikimedia.org/wikipedia/commons/5/53/Wikipedia-logo-en-big.png";
```

Comment:
Don't do that, just concatenate the strings using `~`.

Reply:
Trust me, I tried that. When you concatinate - nothing is being downloaded.

Reply:
Sorry, but what you're saying is not possible. Please check for errors on your side.

If you are confident in that statement, then you need to find the core reason for this. Do not submit a change if you don't fully understand it.

Reply:
Yep, I completely messed up this one.

Reply:
Shall I Close this Pull Request, or try to fix it?

Reply:
Looks OK now. Just the `http` redeclaration issue remains.